moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -221,12 +221,21 @@ struct CollisionRequest
   bool verbose;
 };
 
+enum DistanceRequestType
```

Comment:
Namespaceify your enums, e.g.:   ``namespace DistanceRequestType {``

"This prevents enums from polluting the namespace they're inside. Individual items within the enum are referenced by: Choices::Choice1, but the typedef still allows declaration of the Choice enum without the namespace."
http://wiki.ros.org/CppStyleGuide

Reply:
I will fix it.